lib/src/screens/contact/contact_list_page.dart

```diff
@@ -60,11 +62,13 @@ class ContactListPage extends BasePage {
 
   @override
   Widget body(BuildContext context) {
+    final contacts =
+    contactListViewModel.getContacts(isEditable, selectedCurrency);
+    final walletContacts =
+    contactListViewModel.getWallets(isEditable, selectedCurrency);
     return Container(
         padding: EdgeInsets.only(top: 20.0, bottom: 20.0),
-        child: Observer(
-          builder: (_) {
-            return CollapsibleSectionList(
```

Comment:
There is a bug caused by removing this `Observer` try deleting a contact and it will still be there until you refresh the address book screen.
`Observer` should still be there and you can define `contacts` and `walletContacts` inside the observer so they are refreshed too or if you have another better implementation

Reply:
yes `Observer` should be there, PR updated manually possibly removed by mistake.